web/services/wallet_services.py

```
@@ -0,0 +1,23 @@
+    try:
+        # Generate a secure BIP-39 mnemonic phrase
+        mnemo = Mnemonic("english")
+        mnemonic_phrase = mnemo.generate(strength=128)
+        # Convert the mnemonic to a seed
+        seed = mnemo.to_seed(mnemonic_phrase)
+        # Generate a wallet from the seed
+        account = Account.from_key(seed[:32])
+        return {"mnemonic": mnemonic_phrase, "address": account.address, "private_key": account.key.hex()}
+    except Exception as e:
+        # Log the error (consider using a proper logging mechanism)
+        print(f"Error creating wallet: {str(e)}")
```

Ruff (0.8.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)